generators/2.0/java/templates/spring/gif.ftl

```diff
@@ -0,0 +1,12 @@
+<messageML>
```

Comment:
This should not be for Spring only. We need to provide the same example for Java and Spring generated projects.

Reply:
I will create another task for it. Is it okay?

Reply:
I'd fix everything as part of this PR, it's not really long actually.

Reply:
But if you prefer I can quickly do it once this PR merged

Reply:
I can do try to do it after I finish the unittest. Don't worry 😄